cmd/ingester/app/flags.go

```diff
@@ -111,6 +113,10 @@ func AddFlags(flagSet *flag.FlagSet) {
 		KafkaConsumerConfigPrefix+SuffixEncoding,
 		DefaultEncoding,
 		fmt.Sprintf(`The encoding of spans ("%s") consumed from kafka`, strings.Join(kafka.AllEncodings, "\", \"")))
+	flagSet.String(
+		KafkaConsumerConfigPrefix+SuffixRackID,
+		"",
+		"Configure the 'rack' in which the consumer resides to enable - must be supported by kafka server")
```

Comment:
Incomplete sentence

Reply:
Hi @yurishkuro.
i update the description